ntia_conformance_checker/sbom_checker.py

```diff
@@ -74,16 +74,19 @@ def get_components_without_names(self):
                 components_without_names.append(package.spdx_id)
         return components_without_names
 
-    def get_components_without_versions(self):
-        """Retrieve SPDX ID of components without names."""
+    def get_components_without_versions(self, returnTuples=False):
```

Comment:
See my question in the original issue about whether the option should be `returnTuples` or something more like `returnSDPXIDs`. If you think `returnTuples` is indeed the right way to go, then this code looks good to me.

Reply:
For my use case the tuples would be preferable. I added a longer explanation to [here](https://github.com/spdx/ntia-conformance-checker/issues/168#issuecomment-1870969908)